Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/NativeArrayTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using NUnit.Framework;
+using Unity.Collections;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class NativeArrayTests : TypeTesterBase
+    {
+        public static (int[] array, string serialized)[] representations = new[] {
+            (new int[]{ 1,2,3,4 }, @"[1,2,3,4]"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)